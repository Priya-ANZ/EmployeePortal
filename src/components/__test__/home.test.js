import { render, screen, cleanup } from '@testing-library/react'
import renderer from 'react-test-renderer';
import Home from '../home';

afterEach(() => {
    cleanup();
});

test('test whether sidebar component is present',() =>{
    render(<Home/>);
    const sidebarElement = screen.getByTestId('sidebar');
    expect(sidebarElement).toBeInTheDocument();
})

test('test whether Employee List component is present',() =>{
    render(<Home/>);
    const sidebarElement = screen.getByTestId('employeeList');
    expect(sidebarElement).toBeInTheDocument();
})



test('matches snapshot',() => {
const tree = renderer.create(<Home></Home>).toJSON();
expect(tree).toMatchSnapshot();
})