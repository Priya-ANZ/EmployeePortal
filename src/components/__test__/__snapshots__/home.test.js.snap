// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  classes="main-content"
>
  <div
    className="row"
  >
    <div
      className="col-3"
      data-testid="sidebar"
    >
      <div
        className="accordion"
      >
        <h4
          className="font-text"
        >
          Departments
        </h4>
        <br />
        <div
          className="accordion-item"
        />
      </div>
    </div>
    <div
      className="col"
      data-testid="employeeList"
    />
  </div>
</div>
`;
